src/axom/CMakeLists.txt

```diff
@@ -38,6 +38,11 @@ else()
 endif()
 axom_add_component(COMPONENT_NAME quest  DEFAULT_STATE ${AXOM_ENABLE_ALL_COMPONENTS})
 
+if(ENABLE_CUDA)
+    # Force "early" device linking to avoid propagating an NVCC link dependency
+    set_target_properties(core ${AXOM_COMPONENTS_ENABLED} PROPERTIES CUDA_RESOLVE_DEVICE_SYMBOLS ON)
```

Comment:
Should this go into BLT?

Reply:
Apparently I need to read this PR as well (https://github.com/LLNL/blt/pull/449)... interconnected PRs are  too much before my caffiene has kicked in.